src/guides/sharecode.md

```diff
@@ -0,0 +1,13 @@
+# Share code
+*"Share code"* is a feature which let's users to share their all bot codes to an another bot.
+
+## Creating a share code
+Here is how you can create your own bot share code -
+- Select your bot in BDFD app homepage.
+- Head over to *"Bot Settings"* tab.
+- In *"Share code"* section, click *"Create a share code"* button.
+- Then, click "Generate share code" button and copy the code.
+
+Now, you can give the code to your friend or someone and they will have your bot codes in their bot.
```

Comment:
Should be `or someone other, and they will be able to create a carbon copy of your bot`

```suggestion
Now, you can give the code to your friend or someone other, and they will be able to create a carbon copy of your bot.
```